docs/source/installation/guide_complete/index.rst

```diff
@@ -0,0 +1,363 @@
+.. _installation:guide-complete:
+
+===========================
+Complete installation guide
+===========================
+
+The :ref:`quick installation guide <installation:guide-quick>` is designed to make the setup as simple and portable as possible.
+However, the resulting setup has some :ref:`limitations <installation:guide-quick:limitations>` concerning the available functionality and performance.
+This guide provides detailed information and instructions to set up a feature-complete and performant installation.
+
+To get a working installation of AiiDA, the following steps are required:
+
+#. :ref:`Install the Python Package <installation:guide-complete:python-package>`
+#. :ref:`Create a profile <installation:guide-complete:create-profile>`
+
+
+.. _installation:guide-complete:python-package:
+
+Install the Python Package
+======================
+
+.. important::
+    AiiDA requires a recent version of Python.
+    Please refer to the `Python Package Index (PyPI) <https://pypi.org/project/aiida-core/>`_ for the minimum required version.
+
+To install AiiDA, the ``aiida-core`` Python package needs to be installed which can be done in a number of ways:
+
+
+.. tab-set::
+
+    .. tab-item:: pip
+
+        Installing ``aiida-core`` from PyPI.
+
+        #. Install `pip <https://pip.pypa.io/en/stable/installation/>`_.
+        #. Install ``aiida-core``:
+
+           .. code-block:: console
+
+               pip install aiida-core
+
+    .. tab-item:: conda
+
+        Installing ``aiida-core`` using Conda.
+
+        #. Install `conda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_.
+
+        #. Create an environment and install ``aiida-core``:
+
+           .. code-block:: console
+
+               conda create -n aiida -c conda-forge aiida-core
```

Comment:
```suggestion
        #. Install `conda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_ or `mamba <https://mamba.readthedocs.io/en/stable/installation/mamba-installation.html>`. You might experience that `conda` needs a lot of time to resolve the dependencies. We therefore recommend `mamba` for the installation.

        #. Create an environment and install ``aiida-core``:

           .. code-block:: console

               conda create -n aiida -c conda-forge aiida-core
               # or
               mamba create -n aiida -c conda-forge aiida-core
               
```
I think we should at least add a sentence that the user should consider to use mamba, since resolving the dependencies with conda takes like 5 mins. That is not bearable for a new user.

Reply:
I see the point, but is that our problem really? Should we add another option to tell users to otherwise use [`micromamba`](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) because it takes less space? In my mind, these variations are not our responsability and listing them will make things more difficult to understand than is needed.

Reply:
I think a 5 minutes waiting time is not just an inconvenience but goes into the bug category. My behavior as a user was to stop it, because I thought there was a bug since I don't encounter so long resolving times often. If the package would take 20GB and with micromamba just 1GB would be also worth a mention it.

Reply:
I agree that the slowness of conda is very inconvenient and could be considered a bug, but that is a bug in _conda_ surely and not AiiDA. Anyway, the mamba solver was integrated to conda in 2022 and has been the [default solver since Oct 2023](https://conda.org/blog/2023-11-06-conda-23-10-0-release/#with-this-23100-release-we-are-changing-the-default-solver-of-conda-to-conda-libmamba-solver--). So I would say that people that install conda now won't need mamba

Reply:
Ah thanks for the information! I would say AiiDA has still the reputation to be not installable with conda (No one here at PSI mentioned to me that I just need to update conda when I mentioned this problem), so do you  think it is still worth to make a note about updating one's conda version if one experience slowdowns?

```suggestion
        #. Install `conda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_.

        #. Create an environment and install ``aiida-core``:

           .. code-block:: console

               conda create -n aiida -c conda-forge aiida-core
                
           .. note::

               With `conda` version 23.10.0 the solver that introduced huge slowdowns has been updated.
               If you experience a long resolving time during installation, please consider updating `conda`.
```